src/reanimated2/animation/repeat.ts

```diff
@@ -34,7 +36,14 @@ export const withRepeat = function <T extends AnimationObject>(
           ? _nextAnimation()
           : _nextAnimation;
 
+      const isEvenNumberOfReps =
+        reverse && (numberOfReps <= 0 || numberOfReps % 2 === 0);
```

Comment:
I know that I suggested renaming this variable but actually the name doesn't represent the exact logic here. What about simply removing this variable and moving the condition to the conditional in line 43?

Reply:
This value is used also in the line 106, because it is needed in both `onFrame` and `onStart`. But I can do `animation.onFrame = () => {return true;}` when the condition is met in `onStart`. Then we can inline this value and retain the same logic.